src/commands/play.rs

```diff
@@ -40,6 +42,12 @@ pub async fn _play(
         .as_str()
         .unwrap();
 
+    let flag = match subcommand_args.name.as_str() {
+        "next" => &PlayFlag::NEXT,
+        "all" => &PlayFlag::ALL,
+        _ => flag,
+    };
```

Comment:
make the default case return the `PlayFlag::END` maybe (and for clarity add a "end" => PlayFlag::END" too)

Reply:
Yeah, I should do that. For some reason I thought I should allow for a case where the passed flag takes priority, in case some command decides to wrap the play command. In hindsight that functionality should be removed as the play command is now very dependent on message context.

Reply:
@joao-conde something I forgot about this is that it currently covers the '&_' case. Modifying this to specifically include the "end" case will make the pattern non-exhaustive.

Reply:
Ye, use both, that's what I meant, just for clarity, like

```rust
"end" => PlayMode::End,
_ => PlayMode::End
```